benchmark/micro/append_mix.cpp

```diff
@@ -0,0 +1,93 @@
+		else                                                                                                           \
+			state->conn.Query("create table IF NOT EXISTS test(id INTEGER primary key, area CHAR(6), age TINYINT not " \
+			                  "null, active TINYINT not null);");                                                      \
+	}                                                                                                                  \
+	int32_t get_random_age() {                                                                                         \
+		/* 5, 10, 15 */                                                                                                \
+		return 5 + 5 * random.NextRandom(0, 3);                                                                        \
+	}                                                                                                                  \
+	bool get_random_bool() {                                                                                           \
+		return random.NextRandom() > 0.5;                                                                              \
+	}                                                                                                                  \
+	int32_t get_random_active() {                                                                                      \
+		return int32_t(random.NextRandom(0, 2));                                                                       \
+	}                                                                                                                  \
+	void get_random_area_code(char *area_code) {                                                                       \
+		uint32_t code = uint32_t(random.NextRandom(0, 999999));                                                        \
+		auto endptr = area_code + 6;                                                                                   \
+		endptr = NumericHelper::FormatUnsigned(code, endptr);                                                          \
+		while (endptr > area_code) {                                                                                   \
+			*endptr = '0';                                                                                             \
+			endptr--;                                                                                                  \
+		}                                                                                                              \
+	}                                                                                                                  \
+	void RunBenchmark(DuckDBBenchmarkState *state) override {                                                          \
+		state->conn.Query("BEGIN QUERY");                                                                              \
```

Comment:
Why I can't use `begin query` in duckdb shell?
```
D begin query;
Error: Parser Error: syntax error at or near "query"
LINE 1: begin query;
              ^
D begin transaction;
```

Reply:
Nice catch, it is actually incorrect but the result was not checked.